src/data/index.ts

```diff
@@ -375,6 +376,12 @@ const json: RequirementsJson = {
     },
   },
   minor: {
+    AEM: {
+      name: 'Applied Economics and Management',
+      schools: ['AG', 'BU'],
```

Comment:
shouldn't every college except dyson be allowed to do the minor?

Reply:
@zachary0kent am I wrong about how this works

Reply:
i know that some colleges like engineering ahve their own version of their AEM minor

Reply:
I believe this refers to which college offers the minor

Reply:
where do we store which colleges are eligible? @zachary0kent

Reply:
I don't believe we do? I'm not of such restrictions typically existing anyways.